Mathlib/NumberTheory/RamificationInertia/Galois.lean

```diff
@@ -148,7 +148,7 @@ theorem ramificationIdx_eq_of_isGalois [IsGalois K L] :
   exact (ramificationIdx_map_eq p P σ).symm
 
 /-- All the `inertiaDeg` over a fixed maximal ideal are the same. -/
-theorem inertiaDeg_eq_of_isGalois [p.IsMaximal] [IsGalois K L] :
+theorem inertiaDeg_eq_of_isGalois [IsGalois K L] :
```

Comment:
You should change the docstring too (BTW, it's great that you could remove the `IsMaximal` hypothesis)